handlers/common.go

```diff
@@ -785,3 +781,36 @@ func getExecutionChartData(indices []uint64, currency string, lowerBoundDay uint
 
 	return chartData, nil
 }
+
+func GetTotalWithdrawalsCount(validators []uint64) (uint64, error) {
+	var count uint64
+	validatorFilter := pq.Array(validators)
+	lastExportedDay, err := services.LatestExportedStatisticDay()
+	if err != nil {
+		return 0, fmt.Errorf("error getting latest exported statistic day for withdrawals count: %w", err)
+	}
+	cutoffSlot := (((lastExportedDay+1)*utils.EpochsPerDay() - 1) * utils.Config.Chain.Config.SlotsPerEpoch) + 1
```

Comment:
better to use `utils.GetFirstAndLastEpochForDay(day)` for that

Reply:
fixed